<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FIXWATER</class>
 <widget class="QWidget" name="FIXWATER">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>72</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:0px;color: rgb(246, 184, 75);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="id_key">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设备：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="id_value">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="t_key">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>温度：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="t_value">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="z_key">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>浊度：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="z_value">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ph_key">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>pH：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="ph_value">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
